engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -189,7 +189,7 @@ <h4 translate>Basic Information</h4>
         <span class="info-label" translate>Download Policy</span>
         <span class="info-value"
               bst-edit-select="downloadPolicyDisplay(repository.download_policy)"
-              readonly="denied('edit_products', product)"
+              readonly="!(BastionConfig.enable_lazy_sync) || denied('edit_products', product)"
```

Comment:
Same here -- attach this to a variable in the controller so we are not binding this view to the service.